<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>868</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="settingsWindowLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QFormLayout" name="mainSettingsLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="languageLabel">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="languageLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="languageCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="languageSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="styleLabel">
       <property name="text">
        <string>Visual style</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="styleLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="styleCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="styleSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="generalSettingsBox">
     <property name="title">
      <string>General settings</string>
     </property>
     <layout class="QVBoxLayout" name="generalSettingsBoxLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="confirmDeleteCheckbox">
        <property name="text">
         <string>Ask for confirmation before deleting entries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="confirmCancelCheckbox">
        <property name="text">
         <string>Ask for confirmation before discarding unsaved changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="warnAboutDuplicateNamesCheckbox">
        <property name="text">
         <string>Warn when saving an item with a duplicate name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="onlyPrepareActiveTableCheckbox">
        <property name="text">
         <string>Only prepare active table on startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="openProjectSettingsOnNewDatabaseCheckbox">
        <property name="text">
         <string>Open project settings window when creating a new database</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rememberBox">
     <property name="title">
      <string>Remember after restart</string>
     </property>
     <layout class="QVBoxLayout" name="rememberBoxLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QGroupBox" name="rememberWindowsBox">
        <property name="title">
         <string>Windows</string>
        </property>
        <layout class="QGridLayout" name="rememberWindowsBoxLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="rememberWindowGeometryCheckbox">
           <property name="text">
            <string>Positions and sizes</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="rememberWindowPositionsRelativeCheckbox">
           <property name="text">
            <string>Positions relative to main window</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="rememberTablesBox">
        <property name="title">
         <string>Tables</string>
        </property>
        <layout class="QGridLayout" name="rememberTablesBoxLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>5</number>
         </property>
         <item row="2" column="1">
          <widget class="QCheckBox" name="rememberFiltersCheckbox">
           <property name="text">
            <string>Active filters</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="rememberTableCheckbox">
           <property name="text">
            <string>Open table</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="rememberSortingCheckbox">
           <property name="text">
            <string>Sorting</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="rememberColumnWidthsCheckbox">
           <property name="text">
            <string>Column widths</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="rememberColumnOrderCheckbox">
           <property name="text">
            <string>Column order</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="rememberHiddenColumnsCheckbox">
           <property name="text">
            <string>Hidden columns</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="defaultValuesBox">
     <property name="title">
      <string>Default values for new items</string>
     </property>
     <layout class="QVBoxLayout" name="defaultValuesBoxLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QGroupBox" name="ascentSettingsBox">
        <property name="title">
         <string>Ascent</string>
        </property>
        <layout class="QFormLayout" name="ascentSettingsBoxLayout">
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item row="1" column="0">
          <widget class="QCheckBox" name="ascentTimeCheckbox">
           <property name="text">
            <string>Local time</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="ascentElevationGainCheckbox">
           <property name="text">
            <string>Elevation gain</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="ascentDateLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QSpinBox" name="ascentDateDaysInPastSpinner">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ascentDateDaysInPastLabel">
             <property name="text">
              <string>days in the past</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="ascentDateSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="ascentDateCheckbox">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="ascentTimeLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QTimeEdit" name="ascentTimeWidget">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="ascentTimeSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="ascentElevationGainLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QSpinBox" name="ascentElevationGainSpinner">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string notr="true"> m</string>
             </property>
             <property name="maximum">
              <number>9000</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="ascentElevationGainSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="peakTripSettingsBoxesLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QGroupBox" name="peakSettingsBox">
          <property name="title">
           <string>Peak</string>
          </property>
          <layout class="QFormLayout" name="peakSettingsBoxLayout">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <widget class="QCheckBox" name="peakHeightCheckbox">
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="peakHeightLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QSpinBox" name="peakHeightSpinner">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string notr="true"> m</string>
               </property>
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>9000</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="peakHeightSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="tripSettingsBox">
          <property name="title">
           <string>Trip</string>
          </property>
          <layout class="QFormLayout" name="tripSettingsBoxLayout">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <widget class="QCheckBox" name="tripDatesCheckbox">
             <property name="text">
              <string>Specify dates</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="tripDatesSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ascentViewerBox">
     <property name="title">
      <string>Ascent Viewer</string>
     </property>
     <layout class="QVBoxLayout" name="ascentViewerBoxLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ascentTitleUnderPeakNameCheckbox">
        <property name="text">
         <string>Show ascent title below peak name, not in ascent box</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bottomButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
